spec/lib/pdf_filler/nj1040_pdf_spec.rb

```diff
@@ -1057,6 +1057,33 @@
         expect(pdf_fields["Text41"]).to eq "0"
       end
     end
+
+
+    describe "line 64 child and dependent care credit" do
+      let(:intake) {
+        create(:state_file_nj_intake, :df_data_one_dep, :fed_credit_for_child_and_dependent_care)
+      }
+      let(:submission) {
+        create :efile_submission, tax_return: nil, data_source: intake
+      }
+      it "adds 40% of federal credit for an income of 60k or less" do
+        digits_in_pdf = ""
+        digit_fields = [
+          'undefined_168',
+          'Text192',
+          'Text193',
+          'Text194',
+          'Text195',
+          'Text196',
+        ]
+        digit_fields.each do |field_name, _i|
+          digits_in_pdf << pdf_fields[field_name]
+          digits_in_pdf << "." if field_name == digit_fields[-3]
+        end
+        tax_credit = digits_in_pdf.to_f
+        expect(tax_credit).to eq 400
```

Comment:
[boulder] I am not a big fan of this testing approach. Because the amount of logic that is written here, it feels like the test itself needs a test to confirm that it properly constructs a number out of independent digit strings in the PDF. Philosophically, I would strongly prefer to test the data as it appears in the PDF, rather than constructing a new third-party data type from the object under test, and testing that data type. I think it gives a lot more confidence if we just test the output of the object under test.

You could pull in the refactor work I already started on the `nj-73-refactor-insert-digits-pdf` branch here, to simplify these expect statements, if that still feels important. 

However, I've been thinking about it more, and I'm not entirely sure what's to be gained by the refactor at all - either the one in the code here OR the one on my branch there. It's not as if it's fewer lines of code or something. What are we really gaining in test readability? And is it worth it for what we're losing in test confidence and directness?

Marking this as a boulder because I just want to talk about it as a team as we pick a path forward, and that's kind of what I mean by re-review